generator/src/test/java/org/apache/kafka/message/MessageDataGeneratorTest.java

```diff
@@ -62,7 +62,7 @@ public void testInvalidNullDefaultForInt() throws Exception {
             "  ]",
             "}")), MessageSpec.class);
         try {
-            new MessageDataGenerator().generate(testMessageSpec);
+            new MessageDataGenerator("test").generate(testMessageSpec);
```

Comment:
This should be `org.apache.kafka.common.message` to match the directory that the generated source files are outputted to